addons/l10n_dz/models/invoice.py

```diff
@@ -0,0 +1,9 @@
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    @api.depends('currency_id', 'amount_total')
+    def get_amount2words(self):
```

Comment:
You can probably make a real compute field like that:
https://github.com/odoo/odoo/blob/14.0/addons/l10n_in/models/account_invoice.py#L11

And it could be interesting to make something more generic in master to handle this case.